cmd/nodecmd/destroy.go

```diff
@@ -120,10 +115,18 @@ func destroyNodes(_ *cobra.Command, args []string) error {
 		nodesToStop = append(nodesToStop, monitoringNode)
 	}
 	// stop all load test nodes if specified
-	ltHostsToStop, err := getClusterLoadTestNodes(clusterName)
+	ltHosts, err := getLoadTestInstancesInCluster(clusterName)
```

Comment:
nit: it seems like this could just return the instances? am I missing something?